src/python_testing/TC_DRLK_2_13.py

```diff
@@ -0,0 +1,663 @@
+        self.step("1a")
+        self.fabric_idx1 = await self.read_attributes_from_dut(endpoint=self.common_cluster_endpoint,
+                                                               cluster=Clusters.Objects.OperationalCredentials,
+                                                               attribute=Clusters.OperationalCredentials.Attributes.CurrentFabricIndex
+                                                               )
+        self.step("1b")
+        if self.pics_guard(self.check_pics("DRLK.S.C1d.Rsp")):
+            await self.send_clear_user_cmd(user_index=int(0xFFFE))
+
+        self.step("1c")
+        if self.pics_guard(self.check_pics("DRLK.S.C26.Rsp")):
+            await self.clear_credentials_cmd(credential=NullValue)
+
+        # step 2
+        self.step("2a")
+        if self.pics_guard(self.check_pics("DRLK.S.C26.Rsp")):
+            await self.send_clear_aliro_reader_config_cmd()
+        self.step("2b")
+        await self.send_set_aliro_reader_config_cmd(use_group_resolving_key=False, expected_status=Status.Success)
+        self.step("2c")
+        await self.send_set_aliro_reader_config_cmd(use_group_resolving_key=True, expected_status=Status.Success)
+        # step 3,4,5
+        aliro_attribute_verify_steps = [
+            {"th_step": "3", "cluster": Clusters.Objects.DoorLock,
+             "attribute": Clusters.DoorLock.Attributes.AliroReaderVerificationKey,
+             "aliro_attribute_value": self.verificationKey},
+            {"th_step": "4", "cluster": Clusters.Objects.DoorLock,
+             "attribute": Clusters.DoorLock.Attributes.AliroReaderGroupIdentifier,
+             "aliro_attribute_value": self.groupIdentifier},
+            {"th_step": "5", "cluster": Clusters.Objects.DoorLock,
+             "attribute": Clusters.DoorLock.Attributes.AliroGroupResolvingKey,
+             "aliro_attribute_value": self.groupResolvingKey}
+        ]
+        list_of_verification_objects = [AliroAttributeVerify(**aliro_object) for aliro_object in
+                                        aliro_attribute_verify_steps]
+        await self.aliro_attribute_verifiers(aliro_attribute_verify_steps=list_of_verification_objects)
+
+        #  step 6
+        self.step("6a")
+        await self.send_set_aliro_reader_config_cmd(use_group_resolving_key=False,
+                                                    expected_status=Status.InvalidInState)
+        self.step("6b")
+        await self.send_set_aliro_reader_config_cmd(use_group_resolving_key=True, expected_status=Status.InvalidInState)
+        #  step 7
+        self.step("7")
+        if self.check_pics("DRLK.S.C26.Rsp"):
+            await self.send_clear_aliro_reader_config_cmd()
+
+        # steps 8,9,10
+        aliro_attribute_verify_steps = [
+            {"th_step": "8", "cluster": Clusters.Objects.DoorLock,
+             "attribute": Clusters.DoorLock.Attributes.AliroReaderVerificationKey, "aliro_attribute_value": NullValue},
```

Comment:
```suggestion
            AliroAttributeVerify(
                th_step="8", 
                attribute=Clusters.DoorLock.Attributes.AliroReaderVerificationKey, 
                attribute_value=NullValue),
```

Please make the similar change throught. Note that I dropped the cluster (redundant) and renamed the value (stutters/smurf naming)

Reply:
object will be created in the list